milestones.nix

```diff
@@ -24,8 +24,8 @@
   # Commits sourced from:
   # https://raw.githubusercontent.com/FuelLabs/fuelup/gh-pages/channel-fuel-testnet.toml
```

Comment:
When does that webpage get updated? Look like neither the before or after commits are listed

Reply:
These are done against the gh_pages branch in fuelup. Here is a PR where that occurred for reference https://github.com/FuelLabs/fuelup/pull/657